### PR TITLE
feat: update antgroup data

### PR DESCRIPTION
**Pull request by bytemain**

### CODE REVIEW

**Comment by tyn1998** [comment]
Hello @bytemain 😄

**Comment by bytemain** [comment]
@tyn1998 Wow, Nice to meet you here. This is an interesting project~

File: labeled_data/companies/alibaba/index.yml [hunk 1]
```diff
@@ -50,5 +50,4 @@ data:
     - 102641885 # org:alibaba-open-source-website
     - 101698007 # org:koordinator-sh
     - 100843567 # org:opensergo
-    - 90233428 # org:opensumi
```
Comment (frank-zsy) [inline labeled_data/companies/alibaba/index.yml:53]: Thanks for the data update, are you sure to remove opensumi from alibaba? We can add opensumi to both alibaba and antgroup like seata.

Comment (bytemain) [reply labeled_data/companies/alibaba/index.yml:53]: CleanShot 2022-09-29 at 23 10 25@2x

Maybe it's my fault, so I need add opensumi to the label field?

Comment (frank-zsy) [reply labeled_data/companies/alibaba/index.yml:53]: Yes, you should add `opensumi` to `alibaba/index.yml`'s label field like other projects.

**Comment by frank-zsy** [comment]
LGTM, thanks for the PR.

/approve

**Pull request merged**